collect_app/src/test/java/org/odk/collect/android/projects/ProjectDetailsCreatorTest.kt

```diff
@@ -12,21 +12,35 @@ import org.odk.collect.projects.Project
 @RunWith(AndroidJUnit4::class)
 class ProjectDetailsCreatorTest {
 
-    val projectDetailsCreator = ProjectDetailsCreator(ApplicationProvider.getApplicationContext())
+    private val projectDetailsCreator = ProjectDetailsCreator(ApplicationProvider.getApplicationContext())
```

Comment:
This test wouldn't need Robolectric if the context didn't have to be passed in here, right?

Reply:
Right.